--- exiv2-0.17.1/src/value.cpp.orig	2009-11-04 17:59:10.221228293 +0100
+++ exiv2-0.17.1/src/value.cpp	2009-11-04 17:59:20.220541063 +0100
@@ -42,6 +42,7 @@
 #include <sstream>
 #include <cassert>
 #include <cstring>
+#include <cstdio>
 #include <ctime>
 #include <cstdlib>
 #include <ctype.h>
--- exiv2-0.17.1/xmpsdk/src/XMPMeta.cpp.orig	2009-11-04 17:57:54.726416880 +0100
+++ exiv2-0.17.1/xmpsdk/src/XMPMeta.cpp	2009-11-04 17:58:15.225008059 +0100
@@ -20,6 +20,7 @@
 #include "UnicodeInlines.incl_cpp"
 #include "UnicodeConversions.hpp"
 
+#include <cstdio>
 #if XMP_DebugBuild
 	#include <iostream>
 #endif
